src/test/java/org/opentripplanner/routing/algorithm/mapping/__snapshots__/BikeRentalSnapshotTest.snap

```diff
@@ -296,7 +296,7 @@ org.opentripplanner.routing.algorithm.mapping.BikeRentalSnapshotTest.accessBikeR
         {
           "agencyId": "prt:prt",
           "agencyName": "TriMet",
-          "agencyTimeZoneOffset": 0,
+          "agencyTimeZoneOffset": -25200000,
```

Comment:
Is this expected?

Reply:
It is set here erronously https://github.com/opentripplanner/OpenTripPlanner/blob/c640517d0d501ed02dec6d1e6ade1c5c462031ba/src/main/java/org/opentripplanner/routing/algorithm/mapping/GraphPathToItineraryMapper.java#L285-L286